regression/cbmc/r_w_ok5/main.c

```diff
@@ -0,0 +1,13 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void main()
+{
+  char c[2];
+  assert(__CPROVER_r_ok(c, 3));
+  assert(!__CPROVER_r_ok(c, 3));
```

Comment:
Might be worth checking for valid reads too

```suggestion
  char c[2];
  assert(__CPROVER_r_ok(c, 3));
  assert(!__CPROVER_r_ok(c, 3));
  assert(__CPROVER_r_ok(c, 2)); // Success
  assert(!__CPROVER_r_ok(c, 2)); // Success
```

Reply:
Added (`assert(!__CPROVER_r_ok(c, 2))` does fail though)

Reply:
:man_facepalming: